PhysicsTools/HepMCCandAlgos/python/genPUProtons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+genPUProtons = cms.EDProducer("GenPUProtonProducer",
+	mix = cms.string("mix"),
+	bunchCrossingList = cms.vint32(0),
+	minPz = cms.double( 4800. )
```

Comment:
You are right and I can change the value of "minPz" so that it is relevant for lower-energy simulated events.